spec/test_app_templates/Gemfile.extra

```diff
@@ -1,7 +1,7 @@
 # placeholder to use for pinning against specific gem commit references
 gem 'hydra-pcdm', github:'projecthydra/hydra-pcdm', branch: 'master'
 gem 'hydra-works', github:'projecthydra/hydra-works', branch: 'master'
-gem 'active-fedora', '11.0.0.rc7'
+gem 'active-fedora', '>= 10.3.0.rc1'
```

Comment:
Why do we need this in `Gemfile.extra`? Isn't this dependency already expressed in `curation_concerns.gemspec`?